app/views/employment/MultipleEmploymentsSummaryView.scala.html

```diff
@@ -142,45 +51,134 @@
 
 @layout(pageTitle = headingForUse, taxYear = Some(taxYear), isAgent = user.isAgent) {
 
-    @errorSummary(form.errors.distinct)
-
-    @heading(headingForUse, Some(messages("employment.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-2")
-    <p class="govuk-body">@messages(s"employment.employmentSummary.content.${if(user.isAgent) "agent" else "individual"}")</p>
+    @heading(headingForUse, Some(messages("payeEmployment.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-2")
     @if(isInYear) {
         @{
             govukInsetText(InsetText(
             content = Text(messages(s"employment.employmentInfo.p1.${if(user.isAgent) "agent" else "individual"}", taxYear.toString))
             ))
         }
     }
-    @employmentList(employments)
-    <h2 class="govuk-label--m">@messages("employment.expenses.label")</h2>
-    <p class="govuk-body">@messages(s"employment.expenses.subheading")</p>
-    @expensesList()
+    <h2 class="govuk-heading-m govuk-!-margin-top-6">@messages("employment.employmentSummary.employers")</h2>
+    <p class="govuk-body">@messages(s"employment.employmentSummary.content.${if(user.isAgent) "agent" else "individual"}")</p>
+
+    @{
+
+        def employmentRow(employmentName: String, viewLink: Call): SummaryListRow = {
```

Comment:
You could have taken this method and the one below to do a bit more work so that we only send what is different. Classes for example are the same for all of those.
I know that there is some work planned for refactoring those so it could be done there.

Reply:
I'm just calling the summaryListRow method from the ViewUtils and passing in whats needed. I thought doing it this way makes it easily readable for whats going on, realistically i could just pass it all in into one line.

Reply:
I was thinking creating a local method and we pass less parameters to avoid some repetitions. We can think about this when the story to merge the templates is played.

Reply:
Yeah i think when we merge them together, a-lot of the methods for creating rows will get merged together too